pkg/telemetryquerylanguage/tql/parser.go

```diff
@@ -96,7 +136,7 @@ func (b *Bytes) Capture(values []string) error {
 type Boolean bool
 
 func (b *Boolean) Capture(values []string) error {
-	*b = values[0] == "true"
+	*b = Boolean(strings.EqualFold(values[0], "true"))
```

Comment:
`EqualFold` compares the strings as case-insensitive right?  I think we do want to force `true` and `false` in the language to be exactly lowercase `true` and `false`.  It looks like the lexer's tokens is enforcing that now, so could `values[0]` ever be anything but `true` and `false`?  If a `True` made it in here that would probably be a bug.

Reply:
Good catch; I had originally wanted to allow SQL-style conditions with all-caps keywords, but that turned out to be troublesome in a few cases, so I backed it out but missed this one.